db/c_test.c

```diff
@@ -527,6 +528,22 @@ int main(int argc, char** argv) {
     }
   }
 
+  StartPhase("statistics");
+  {
+    uint64_t block_cache_miss;
+    uint64_t block_cache_hit;
```

Comment:
It causes unused variable compile warning. Or can you add `__attribute__((__unused__))` to them?